src/main/resources/application.yml

```
@@ -21,4 +21,9 @@ spring:
     init:
       mode: never
   cors:
-    allowed-origins: "*"
\ No newline at end of file
+    allowed-origins: "*"
+
+
+auth:
+  jwt:
+    secret-key: "asdojfo"
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)